### DISCUSSION TITLE
Slash Commands Requirement Coming in April?

### DISCUSSION THREAD

**Discussion by ArahamaMaiko** [jagrosh/MusicBot#1137] [General]
Discord is going to be requiring slash commands coming in April, is there already plans to implement them? I hear they are quite difficult to add.

**Comment by jagrosh** [comment]
Slash commands are not required. Discord is limiting access to message content at the beginning of May, but this only affects bots on more than 75 servers, which isn't supported by JMusicBot.

That being said, JMusicBot _will_ add slash command support eventually, but it is unlikely to happen before the beginning of May.
